src/datasets/arrow_dataset.py

```diff
@@ -3280,6 +3391,7 @@ def train_test_split(
                 If int, represents the absolute number of train samples.
                 If None, the value is automatically set to the complement of the test size.
             shuffle (:obj:`bool`, optional, default `True`): Whether or not to shuffle the data before splitting.
+            stratify (:obj:`str`, optional, default `None`): data is split in a stratified fashion, using the specified class labels
```

Comment:
Better docstring:
```suggestion
            stratify (:obj:`str`, optional, default `None`): The column name of labels to use to stratify data.
```

Let's add the `isinstance(stratify, datasets.ClassLabel)` check to the function body

Reply:
Oops, you are right. I mean this: `isinstance(self.features[stratify], ClassLabel)`

Reply:
Yeah